src/psse_export.jl

```diff
@@ -796,12 +1006,16 @@ function write_to_buffers!(
     branches_with_numbers = get!(exporter.components_cache, "branches") do
         branches = sort!(
             collect(
-                PSY.get_components(Union{PSY.Line, PSY.MonitoredLine}, exporter.system),
+                PSY.get_components(
+                    Union{PSY.Line, PSY.MonitoredLine, PSY.DiscreteControlledACBranch},
+                    exporter.system,
+                ),
             );
             by = branch_to_bus_numbers,
         )
         [(branch, branch_to_bus_numbers(branch)) for branch in branches]
```

Comment:
this code is pretty crazy, please implement it in a function with adequate comments.